packages/runtime/test/anonymous/tokens.test.ts

```diff
@@ -1,13 +1,14 @@
 import { TokenDTO, TransportServices } from "../../src";
-import { NoLoginTestRuntime, RuntimeServiceProvider, TestRuntime, uploadOwnToken } from "../lib";
+import { NoLoginTestRuntime, RuntimeServiceProvider, TestRuntime, TestRuntimeServices, uploadOwnToken } from "../lib";
 
 const serviceProvider = new RuntimeServiceProvider();
 let transportServices: TransportServices;
 let noLoginRuntime: TestRuntime;
+let runtimeService: TestRuntimeServices;
 
 beforeAll(async () => {
-    const runtimeServices = await serviceProvider.launch(1);
-    transportServices = runtimeServices[0].transport;
+    runtimeService = (await serviceProvider.launch(1))[0];
+    transportServices = runtimeService.transport;
```

Comment:
you should remove the transportServices if you expose the whole RuntimeServices.